core/ast2100/ast2100.js

```diff
@@ -0,0 +1,516 @@
+"use strict";
+/*global DCTSIZE, DCTSIZE2, JpegHuffmanTable, BitStream, AST2100IDCT, AAN_IDCT_SCALING_FACTORS, ZIGZAG_ORDER */
+/*global ATEN_QT_LUMA, ATEN_QT_CHROMA, TABLE_CLASS_AC, TABLE_CLASS_DC */
+/*global BITS_AC_LUMA, BITS_AC_CHROMA, BITS_DC_LUMA, BITS_DC_CHROMA, HUFFVAL_AC_LUMA, HUFFVAL_AC_CHROMA, HUFFVAL_DC_LUMA, HUFFVAL_DC_CHROMA */
+/*global inRangeIncl, isEmpty */
+/*global fmt_u8a, fmt_u8, fmt_u16 */
```

Comment:
I'd be happy to; do you have concrete suggestions?  Like I mentioned above, I'm not particularly familiar with noVNC's module system (which didn't exist when these patches were already written), and I don't work in JavaScript at the moment (professionally) so I'm a bit rusty on the finer points of project organization.

Are there choices other than moving everything defined in each file into a single object and then replacing references with e.g. `ast2100const.ATEN_QT_LUMA`, or was that what you had in mind?